src/olympia/amo/templates/amo/emails/locales_completion.ltxt

```diff
@@ -0,0 +1,18 @@
+Hi,
+{% if locales_below_threshold %}
+The following locales are below threshold of {{ COMPLETION_THRESHOLD }}% or completely absent in one of our projects in Pontoon:
+- {{ locales_below_threshold }}
+{% endif %}
+{% if locales_to_keep_despite_being_below_threshold %}
+The following locales are below threshold, but should never be removed from AMO:
+- {{ locales_to_keep_despite_being_below_threshold }}
+{% endif %}
+{% if locales_above_threshold %}
+The following locales are above threshold and not yet enabled:
+- {{ locales_above_threshold }}
+{% endif %}
+{% if not locales_below_threshold and not locales_above_threshold and not locales_to_keep_despite_being_below_threshold %}
```

Comment:
We should just not send the email in this case or? What is the argument for regularly sending an email with no information in it?

Reply:
It's only going to be sent monthly, so it's not going to be super spammy.

I wanted to still send it in this case to let people know everything is working properly. It's a cheap built-in monitoring system :)